subsys/bluetooth/audio/shell/bap_broadcast_assistant.c

```diff
@@ -38,9 +38,14 @@
 
 #define INVALID_BROADCAST_ID 0xFFFFFFFFU
 
+#define MAX_BROADCASTNAME_LEN 32
+#define MIN_BROADCASTNAME_LEN 3
```

Comment:
The assigned number document define these as 4 and 128
```suggestion
#define MAX_BROADCASTNAME_LEN 128
#define MIN_BROADCASTNAME_LEN 4
```

I would suggest to define these as values in `audio.h` instead, as 
```c
#define BT_AUDIO_BROADCAST_NAME_LEN_MIN 4
#define BT_AUDIO_BROADCAST_NAME_LEN_MAX 128
```

Reply:
OK.